netutils/ip.py

```diff
@@ -158,6 +158,39 @@ def ip_subtract(ip: str, val: int) -> str:
     return str(ipaddress.ip_address(ip) - val)
 
 
+def is_classfull(ip_network: str) -> bool:
+    """Determines if a CIDR network address is within unicast class full boundaries.
+
+       The following class boundaries are checked:
+
+       * Class A: 0.0.0.0/8 -> 127.0.0.0/8
+       * Class B: 128.0.0.0/16 -> 191.255.0.0/16
+       * Class C: 192.0.0.0/24 -> 223.255.255.0/24
+
+    Args:
+        ip_network (str): A network string that can be parsed by ipaddress.ip_network.
```

Comment:
```suggestion
        ip_network: A network string that can be parsed by ipaddress.ip_network.
```
since netutils is fully typed this is not required.